lession2.html

```diff
@@ -6,58 +6,338 @@
+    displayCity();
+    function displayCity(){
+        var i;
+        for(i=0;i<states.length;i++)
+        {
+            let opt=document.createElement('option');
+            opt.value=states[i];
+            opt.innerText=states[i];
+            document.getElementById('inputCity').appendChild(opt);
+        }
+    }
+    
+    function display(){
+        var i;
+        for(i=0;i<person.length;i++)
+        {
+            let tRow=document.createElement('tr');
+
+            let td1=document.createElement('td');
+            td1.innerText=person[i].name;
+
+            let td11=document.createElement('td');
+            td11.innerText=person[i].gender;
+            
+            let td2=document.createElement('td');
+            td2.innerText=person[i].phone;
+            
+            let td3=document.createElement('td');
+            td3.innerText=person[i].email;
+
+            let td4=document.createElement('td');
+            td4.innerText=person[i].pass;
+
+            let td5=document.createElement('td');
+            td5.innerText=person[i].birthday;
+
+            let td6=document.createElement('td');
+            td6.innerText=person[i].address;
+
+            let td7=document.createElement('td');
+            td7.innerText=person[i].city;
+
+            let td8=document.createElement('td');
+            td8.innerText=person[i].zip;
+
+            let td9=document.createElement('td');
+            td9.innerText=person[i].intro;
+
+            tRow.appendChild(td1);
+            tRow.appendChild(td11);
+            tRow.appendChild(td2);
+            tRow.appendChild(td3);
+            tRow.appendChild(td4);
+            tRow.appendChild(td5);
+            tRow.appendChild(td6);
+            tRow.appendChild(td7);
+            tRow.appendChild(td8);
+            tRow.appendChild(td9);
+
+            document.getElementById("myBody").appendChild(tRow);
+        }
+    }
+    let name = document.getElementById('inputName');
+    let email= document.getElementById('inputEmail');
+    let phone = document.getElementById('inputPhone');
+        let pass= document.getElementById('inputPassword');
+        let ad= document.getElementById('inputAddress');
+        let ad1= document.getElementById('inputAddress1');
+        let city= document.getElementById('inputCity');
+        let zip= document.getElementById('inputZip');
+        let birthday=document.getElementById('inputBirthday');
+        let intro=document.getElementById('inputIntro');
+
+        
+    function enableButt(){
+        
+        if(email.value != '' && pass.value!= '' && ad.value !='' && ad1.value !='Select State' && city.value !='' &&  zip.value !='' && name.value !='' && phone.value != '' )
+        {
+            
+            document.getElementById("signButt").disabled=false;
+        }
+        else
+        {
+            document.getElementById("signButt").disabled=true;
+        }
+
+
+    }
```

Comment:
```suggestion
function enableButt() {
    if (
        email.value != ''
        && pass.value != ''
        && ad.value != ''
        && ad1.value != 'Select State'
        && city.value != ''
        && zip.value != ''
        && name.value != ''
        && phone.value != ''
    ) {
        document.getElementById("signButt").disabled = false;
    } else {
        document.getElementById("signButt").disabled = true;
    }
}
```
Viết như này cho nó chuẩn convention e ơi